index.html

```diff
@@ -332,7 +332,7 @@ <h2>Blog</h2>
             <div class="col-12 col-lg-4 border-0">
               <div class="card text-white overflow-hidden shadow">
                 <a href="{% link {{post.path}} %}" class="overflow-hidden">
-                  <img src="{% link /assets/img/blog/{{ post.image }} %}" class="card-img" alt="">
+                  <img src="{% link /assets/img/blog/{{ post.image }} %}" class="card-img" alt="Card image">
```

Comment:
We should use the same alt as [in the blog layout](https://github.com/RIOT-OS/riot-os.org/pull/38/files#diff-d3b00176e2ef0283e5270788a27985f94c998cd9f6bdb6af32e2b17fa38ca7adR9) here.

```suggestion
                  <img src="{% link /assets/img/blog/{{ post.image }} %}" class="card-img" alt="Banner for '{{ page.title }}'">
```